src/humanpy/hum_kin1.py

```diff
@@ -0,0 +1,263 @@
+PACKAGE = 'humanpy'
```

Comment:
I renamed them. I hope that now the names are explicative! 
I added a comment to all the functions.